pkg/apis/performance/performance_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/ghodss/yaml"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	v1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1alpha1"
+	v1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1"
```

Comment:
the renamed import (`v1`) seems unneeded now

Reply:
will check, thanks

Reply:
right, is already called v1!